cmd/corectl/main.go

```diff
@@ -68,6 +68,8 @@ var commands = map[string]*command{
 	"get":                  {get},
 	"add":                  {add},
 	"rm":                   {rm},
+	"set":                  {set},
+	"unset":                {unset},
```

Comment:
Oh hm, I guess for sets it needs to take the tuple value to remove. We could start by requiring it no matter what and if it becomes annoying, we can relax that for single-tuple options.